### PR TITLE
Milestone 10:  Form Validation

### PR DESCRIPTION
**Pull request by eroiyo**
Now we added some Validations using JavaScript 🔥 

-Created error class
  *The error class give to the element a red solid border ❌ 
-Created success class
  *the success class give the element a green solid border ✔️ 
  
-Created some function to look if there an invalid information
 *A function that cheek is they are blank 
 *A function that cheek is they are in lowercase or not
 *A function that  cheek is they overpass character limit
 *Depending on the state of the input, depends the class they will get
 
 Thats all!

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -31,6 +31,89 @@ function closeModal() {
...
+    preventOrNot = true;
+  }
+  if (isEmpty(textarea, stextarea, 'Message is required') === false) {
+    preventOrNot = true;
+  } else if (areaCharacterLimit(textarea, stextarea, 'The character should not exceed 500') === false) {
+    preventOrNot = true;
+  }
+  if (preventOrNot) {
+    event.preventDefault();
+  }
```
Comment (Temmarie) [inline js/script.js:118]: - [ ] Good job implementing this requirement. There's a minor change that needs to be done here and you're good to go.
- The error message should show up close to the "submit" button.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @eroiyo     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 
- Your form validations work as expected, I especially like how you organised your JS files ✔️ 
 
 However, there are a few  minor changes required for this project to be approved.
- - -

### **Required Changes** ♻️
- [ ] The vertical scroll bar is missing on your page hereby making it hard to navigate and restricting movement to the use of the keyboard. Kindly fix this.

image

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A
_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Comment by eroiyo** [comment]
The scrollbar was added back!

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @eroiyo,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**